packages/block-library/src/group/edit.js

```diff
@@ -33,14 +38,36 @@ function GroupEdit( { attributes, clientId } ) {
 		}
 	);
 
-	return (
-		<TagName { ...blockProps }>
-			<BoxControlVisualizer
-				values={ attributes.style?.spacing?.padding }
-				showValues={ attributes.style?.visualizers?.padding }
+	const inspectorAdvancedControls = (
+		<InspectorAdvancedControls>
```

Comment:
Any reason for not inlining this in the main `return` call?

Reply:
No reason other than consistency with the template-part block where we do the same.
I inlined it now, and did the same for the template-part :+1: